ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/send/view/SendView.kt

```diff
@@ -360,7 +355,7 @@ private fun SendForm(
             onReferenceClick = goBalances
         )
 
-        Spacer(modifier = Modifier.height(ZcashTheme.dimens.spacingLarge))
+        Spacer(modifier = Modifier.height(24.dp))
```

Comment:
Can we use a value from dimensions instead of a number for spacings here and in several other places changed by this PR? Do we have such a size defined there?

Reply:
we do not have ZashiDimensions tied to these parts of design :(

Reply:
done